INFO     root:api_helper.py:35 Пользователь qa_guru существует